toolkits/global/packages/global-forms/README.md

```diff
@@ -204,6 +204,76 @@ The `name` property is placed at the top level and inherited by each input.
+    "showLabel": false,
+    "imageDescription": "An image of a cartoon face with a sad expression."
+  },
+  {
+    "label": "OK",
+    "value": "OK",
+    "id": "radio-ok",
+    "svg": "<svg xmlns=\"http://www.w3.org/2000/svg\" focusable=\"false\" aria-hidden=\"true\" viewBox=\"0 0 100 100\"><defs><clipPath id=\"aneutral\"><path d=\"M100 0v100H0V0h100Z\"/></clipPath><clipPath id=\"bneutral\"><path d=\"M25.626327 0v4.52229299H0V0h25.626327Z\"/></clipPath><clipPath id=\"cneutral\"><path d=\"M14.8407643 0v4.16135881H0V0h14.8407643Z\"/></clipPath><clipPath id=\"dneutral\"><path d=\"M14.7983015 0c.0849257 1.48619958.1486199 2.80254777.2547771 4.64968153H0V0h14.7983015Z\"/></clipPath><clipPath id=\"eneutral\"><path d=\"M50 0c27.6220807 0 50 22.3991507 50 50s-22.3779193 50-50 50S0 77.6008493 0 50 22.3779193 0 50 0Zm0 4.01273885C24.6072187 4.01273885 4.01273885 24.6072187 4.01273885 50 4.01273885 75.3927813 24.6072187 95.9872611 50 95.9872611c25.3927813 0 45.9872611-20.5944798 45.9872611-45.9872611S75.3927813 4.01273885 50 4.01273885Z\"/></clipPath></defs><g clip-path=\"url(#aneutral)\"><g clip-path=\"url(#bneutral)\" transform=\"translate(37.32484076 67.40976645)\"><path d=\"M0 0h25.626327v4.52229299H0V0z\"/></g><g clip-path=\"url(#cneutral)\" transform=\"translate(69.70276008 42.88747346)\"><path d=\"M0 0h14.8407643v4.16135881H0V0z\"/></g><g clip-path=\"url(#dneutral)\" transform=\"translate(15.45647558 42.6326964)\"><path d=\"M0 0h15.0530786v4.64968153H0V0z\"/></g><g clip-path=\"url(#eneutral)\"><path d=\"M0 0h100v100H0V0z\"/></g></g></svg>",
```

Comment:
Thanks. Yes, I have run all of these SVGs through SVGOMG already. Perhaps it's because I kept the precision setting quite high. I'd have a play and see if I can find some settings and a level of precision that improves the compression.

Reply:
FWIW you can usually go for very low precision without any visible degradation, in my experience.

Reply:
Thanks. Yeah, I'm struggling to remember now but I think I had an issue with the visual appearance of one of the SVGs and upping the precision fixed it. I usually leave it with the default. I'll have a play now and figure out something better.

Reply:
Just had a demo with James A and he'd like me to use the SVGs he's created for the CSAT. So I'll get those from him and will use those on this demo instead (I'll run them through SVGOMG)